frontend/src/components/Rbac.tsx

```diff
@@ -18,7 +18,7 @@ type RbacDropdownProps<T = unknown> = {
 
 type Actions<T = unknown> = {
     id: string
-    text: string
+    text: string | React.ReactNode
```

Comment:
```suggestion
    text: React.ReactNode
```

ReactNode already includes string